lulesh-v1.0/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 ###############################################################################
 #
-# Copyright (c) 2017, Lawrence Livermore National Security, LLC.
+# Copyright (c) 2020, Lawrence Livermore National Security, LLC.
```

Comment:
We should probably keep these as a range of years like in RAJA etc. But, we shouldn't be changing any copyright info in the files for the actual proxy apps. Feel free to take the 'update copyright' script in the RAJA/scripts directory and modify it to make it easy to update copyright dates in the future in this repo.

Reply:
Done